src/integration/spring/src/jvmMain/kotlin/community/flock/wirespec/integration/spring/kotlin/configuration/WebClientConfigurationProperties.kt

```diff
@@ -0,0 +1,8 @@
+package community.flock.wirespec.integration.spring.kotlin.configuration
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "wirespec.spring.webclient")
+data class WebClientConfigurationProperties(
+    var baseUrl: String = "http://localhost:8080",
+)
```

Comment:
IIRC this can be simplified to

```suggestion
@ConfigurationProperties(prefix = "wirespec.spring.webclient")
data class WebClientConfigurationProperties(
    val baseUrl: String,
)
```

making this an immutable value, and removing the default